src/main/scala/tilelink/Monitor.scala

```diff
@@ -161,6 +173,8 @@ class TLMonitor(args: TLMonitorArgs, monitorDir: MonitorDirection = MonitorDirec
     val mask = edge.full_mask(bundle)
     val legal_source = Mux1H(edge.client.find(bundle.source), edge.client.clients.map(c => c.sourceId.start.U)) === bundle.source
 
+    checkFormatOpcode(bundle.opcode, edge, "B")
```

Comment:
I think the manager parameters only applied to traffic on channel A (and C, but I don't think we send any of those transactions on channel C)

Reply:
what are "manager parameters"? Looking at the spec, channels A, B, and C seem to have the same signals?